.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         node_version: [10, 11, 12]
-        os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest]
```

Comment:
Maybe you could rename ci as ci-linux. Afterall, if we merge this PR we would have three files for three OS

Reply:
Done.